.editorconfig

```diff
@@ -0,0 +1,24 @@
+﻿# EditorConfig is awesome:http://EditorConfig.org
```

Comment:
> Which editors understand this?

VS 2017 (no plugins). 
VS Code with plugin.

It's not yet enforced by csc, but IIUC they are planning to add that.